lib/entities/content-type.js

```diff
@@ -223,7 +227,38 @@ function createContentTypeApi (http) {
      * @func isDraft
      * @return {boolean}
      */
-    isDraft: createDraftChecker()
+    isDraft: createDraftChecker(),
+
+    /**
+     * Omits and deletes a field if it exists on the contentType. This is a convenience method which does both operations at once and potentially less
+     * safe than the standard way. See note about deleting fields on the Update method.
+     * @memberof ContentType
+     * @func omitAndDeleteField
+     * @return {Promise<ContentType>} Object returned from the server with updated metadata.
+     */
+    omitAndDeleteField: function (id) {
+      this.changeField(id, 'omitted', true)
```

Comment:
To avoid unnecessary requests, what do you think about adding a check if the field exists or not before calling `this.update`?

Reply:
Maybe throw an error to inform the user that the field does not exist

Reply:
good call on doing a check first!

for throwing an error-- I was mimicking behavior of `update`, where if you try to update a field that doesn't exist it just does nothing. This is the first method we have that operates directly on fields though, so I could see why it makes sense to throw a specific error

Reply:
for the `update` if the fields do not exist the API will create it if it is valid otherwise it will return an error

Reply:
ah I see my error in my test from before. thanks for checking on this